rl_algorithms/common/buffer/distillation_buffer.py

```diff
@@ -13,13 +13,13 @@
 
 class DistillationBuffer:
     """Class for managing reading and writing of distillation data.
-       Distillation data is stored in the buffer_path location on the actual hard disk.
+       Distillation data is stored in the distillation_dataset_path location on the actual hard disk.
        The data collected by the teacher is stored as individual pickle files.
        It is also read as batch through the pytorch DataLoader class.
 
     Attributes:
         batch_size (int): size of batch size from distillation buffer for training
-        buffer_path (list): list of distillation buffer path
+        distillation_dataset_path (list): list of distillation buffer path
```

Comment:
I think `distillation_dataset_path` can be changed `dataset_path` due to it is in `DistillationBuffer`

Reply:
resolved in 661a128